hub-agent/values.yaml

```diff
@@ -24,6 +24,9 @@ controllerDeployment:
     metricsURL: http://traefik-hub.hub-agent.svc.cluster.local:9100/metrics
   args:
     - --log-level=debug
+  # env:
+  #   - name: "LOG_LEVEL"
+  #     value: "debug"
```

Comment:
I think we can remove that from here no ?

Reply:
I left it there to document the available properties, similar to this https://github.com/traefik/hub-helm-chart/blob/452cffb6fb875f4163942d1d6f54dfb58ddc381f/hub-agent/values.yaml#L66-L71

Reply:
It's a little weird for the log level because it's also defined in the args just above